lib/script.js

```diff
@@ -49,6 +49,19 @@ export const config = {
       'Directory of the script',
     ],
   },
+  position: {
+    title: 'Panel position',
+    description: 'Position of the panel with script output. '
+        + '(Changes to this value will be applied upon reopening the panel.)',
```

Comment:
Please use a multi-line string literal 
```
    description: `Position of the panel with script output.
    (Changes to this value will be applied upon reopening the panel.)`,
```

Reply:
I chose to use a simple string literal broken over multiple lines in order to keep the resulting strings on one line when displayed.